src/test/java/com/hotels/beeju/extensions/HiveMetaStoreJUnitExtensionTest.java

```diff
@@ -72,13 +72,11 @@ public void customDbNameAndConfInitialised() throws Exception {
 
   @Test
   public void createExistingDatabase() {
-    Assertions.assertThrows(AlreadyExistsException.class, () -> {
-      hiveDefaultName.createDatabase(hiveDefaultName.databaseName());
-    });
+    Assertions.assertThrows(AlreadyExistsException.class, () -> hiveDefaultName.createDatabase(hiveDefaultName.databaseName()));
```

Comment:
this is probably supposed to be on 2 lines (over 120 characters :( )

Reply:
you are correct :')